remote_config/src/desktop/rest.cc

```diff
@@ -152,72 +149,33 @@ void RemoteConfigREST::SetupRestRequest(
       app_instance_id_token_);  // TODO(cynthiajiang) change to installations
 
   rc_request_.SetPlatformVersion("2");
+  std::string locale = firebase::internal::GetLocale();
+  if (locale.length() != 0) {
+    rc_request_.SetLanguageCode(locale);
+    rc_request_.SetCountryCode(
+        locale.substr(0, 2));  // get country code from locale
```

Comment:
How sure are we that language codes returned here are 2 characters long.  Standard language codes may be longer.  [For instance](https://www.google.com/search?q=locale+country+codes&oq=locale+country+code).

Reply:
Yeah the locale code could be longer, but I thought the country code would always be the first two letters

Reply:
Need to find out a way to grab country code, the current approach is to get the two digit language from locale